samples/WorkerBindingSamples/Blob/BlobInputBindingSamples.cs

```diff
@@ -114,5 +114,20 @@ public BlobInputBindingSamples(ILogger<BlobInputBindingSamples> logger)
 
             return req.CreateResponse(HttpStatusCode.OK);
         }
+
+        [Function(nameof(BlobContainerClientInputFunction))]
+        public HttpResponseData BlobContainerClientInputFunction(
```

Comment:
nit: move to be above the BlobClient sample
nit2: rename to `BlobInputContainerClientFunction` to match the naming pattern of the rest of the functions

Reply:
Changed!